source/common/stats/stats_impl.h

```diff
@@ -23,6 +23,10 @@
 
 #include "absl/strings/string_view.h"
 
+extern "C" {
+#include "circllhist.h"
```

Comment:
Can you add a TODO here to get extern "C" in the headers for the library? (Or just do a PR over there). cc @postwait

Reply:
Fixed upstream.

Reply:
@postwait awesome. I updated the latest version and it is no longer needed.